zeebe/exporters/opensearch-exporter/src/test/java/io/camunda/zeebe/exporter/opensearch/OpensearchExporterIT.java

```diff
@@ -413,14 +441,18 @@ void shouldNotTimeoutWhenUpdatingLifecyclePolicyForExistingIndices() {
       // using 490 here as we will export one more record after (1 main shard, 1 replica)
       final int limit = 490;
       for (int i = 0; i < limit; i++) {
-        final var record = factory.generateRecord(ValueType.JOB);
+        final var record =
+            factory.generateRecord(
+                ValueType.JOB, r -> r.withBrokerVersion(VersionUtil.getVersionLowerCase()));
```

Comment:
🔧 Might make sense to make this the default - that the broker version is set correctly?